ThingiBrowser/PreferencesHelper.py

```diff
@@ -62,3 +71,24 @@ def getSettingValue(cls, setting_name: str) -> str:
         """
         preference_key = "{}/{}".format(Settings.PREFERENCE_KEY_BASE, setting_name)
         return CuraApplication.getInstance().getPreferences().getValue(preference_key)
+
+    @classmethod
+    def getSettingOptions(cls, setting_name: str) -> List[Dict[str, str]]:
+        """
+        Get any selection option for a particular preference
+        :param setting_name: The name of the setting to get the options for.
+        :return: The options for the setting.
+        """
+        options = {
+            Settings.DEFAULT_API_CLIENT: [
```

Comment:
I think it's better to move the options to another key in the correct dict item of getAllSettings.

Reply:
How would I get these options in line 50 then so that it get supplied to the SettingsItem model?

Reply:
Just add a new key "options" or something to it, then the UI can fetch it directly from the model itself.

Reply:
Oh I get what you are saying now. I tried that and then I moved it out for some reason that escapes me right now... I'll move it back then.